templates/part.content.php

```diff
@@ -19,7 +19,7 @@ class="app-content-detail">
     <ul>
         <li class="item {{ ::Content.getFeed(item.feedId).cssClass }}"
             ng-repeat="item in Content.getItems() |
-                orderBy:[Content.orderBy()] track by item.id"
+                orderBy:'id':Content.oldestFirst:Content.sortIds track by item.id"
```

Comment:
Using `orderBy` in a template is covered here: https://docs.angularjs.org/api/ng/filter/orderBy#usage

`'id'` is the field we want to sort by, `Content.oldestFirst` says if we want reverse order or not, and `Content.sortIds` is the new comparator function.